### PR TITLE
Add ability to access chat room(s)

### PR DESCRIPTION
**Pull request by a-stepin**
tl;dr Now, when accessing the URL, you can go from main page -> map selection page -> chat room

*Added page (including styling and basic Javascript functionality) from where the user can enter chat rooms they are part of (so far, this chat room is hard-coded, but in the near future, the chat rooms will be fetched from the chat rooms mapped the to user).

*Added basic styling to the page; Intended result is similar to the first image under User Interface in the design doc (http://go/capstone-chap)

### CODE REVIEW

File: src/main/webapp/main.js [hunk 1]
```diff
@@ -95,5 +94,5 @@ function getServerUrl() {
         protoSpec = 'ws:';
     }
 
-    return protoSpec + "//" + location.host + "/chat/" + defaultChatRoomID + "?idToken=" + idToken;
+    return protoSpec + "//" + location.host + "/chat/" + chatRoomID + "?idToken=" + defaultIDToken;
```
Comment (Felomeng) [inline src/main/webapp/main.js:98]: can room id be null?

File: src/main/webapp/main.js [hunk 2]
```diff
@@ -80,8 +81,6 @@ connection.onmessage = (event) => {
  * @return {string} The server's URL.
   */
 function getServerUrl() {
-    
-    var defaultChatRoomID = "1234goroom";
     var protoSpec;
     var defaultIDToken = 12;
```
Comment (Felomeng) [inline src/main/webapp/main.js:86]: What is it used for?

File: src/main/webapp/mapSelection.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Loads available chat rooms to the user
+  */
+function loadChatRooms() {
```
Comment (Felomeng) [inline src/main/webapp/mapSelection.js:18]: Is it loading only 1 room?

Comment (a-stepin) [reply src/main/webapp/mapSelection.js:18]: For now, we are manually loading a single room, but our intention is that in the coming weeks, we can fully develop functionality to fetch the rooms assigned to the user.

File: src/main/webapp/main.js (hunk 2 above)
Comment (a-stepin) [reply src/main/webapp/main.js:86]: The method is used to return the URL of a chat room server in a format that could be accessed by a WebSocket.

**Review by AliceVlasov — APPROVED** [review]

**Review by Felomeng — APPROVED** [review]

**Pull request merged**